api/src/services/auth/group/group.class.ts

```diff
@@ -0,0 +1,49 @@
+import { KnexAdapterParams } from '@feathersjs/knex'
+import {
+  resolveData,
+  resolveQuery,
+  resolveAll,
+  validateQuery,
+  validateData,
+} from '@feathersjs/schema'
+
+import {
+  GroupData,
+  GroupResult,
+  GroupQuery,
+  groupDataValidator,
+  groupQueryValidator,
+} from './group.schema'
+import { groupResolvers } from './group.resolver'
+import { HookContext } from '../../../declarations'
+import { authenticate } from '@feathersjs/authentication'
+import { ObjectionService } from '../../../feathers-objection'
+import { UserResult } from '../../auth/user/user.schema'
+import { USER_PROFILE } from '@locokit/definitions'
+import { Forbidden } from '@feathersjs/errors/lib'
+
+export const groupHooks = {
+  around: {
+    all: [authenticate('jwt'), validateData(groupDataValidator), resolveAll(groupResolvers)],
```

Comment:
Maybe `authenticate('jwt') ` can be put only in `before`

Reply:
I prefer check the user is authenticated before validating / resolving data.